src/javascript/app_2/Stores/Modules/Trading/Helpers/proposal.js

```diff
@@ -36,11 +36,14 @@ export const createProposalRequests = (store) => {
 
 const createProposalRequestForContract = (store, type_of_contract) => {
     const obj_expiry = {};
+
     if (store.expiry_type === 'endtime') {
+        const server_time = store.root_store.common.server_time;
+        const start_date  = moment.unix(store.start_date || (server_time / 1000)).utc();
```

Comment:
does server_time/1000 need to be .toFixed(0)?

Reply:
since `store.root_store.common.server_time` is a moment object it doesn't need to divide it by 1000.